_includes/component-changelog.html

```diff
@@ -0,0 +1,9 @@
+{% assign componentKey = componentName | slugify %}
+
+<h2 class="site-component-section-title" id="changelog-{{ componentKey }}">Latest updates</h2>
+<p>All code changes for this component are tracked in its <a href="https://github.com">CHANGELOG.md</a>.</p>
+<p>Meaningful guidance updates are listed below:</p>
+<ul class="usa-content-list">
+    <li>Aug. 5, 2022 - <a href="https://github.com/uswds/uswds-site/pull/1737">Added type="button" to the component code and guidance.</a></li>
+    <li>July 20, 2022 - <a href="https://github.com/uswds/uswds-site/pull/1718">Update documentation to use the correct multi-select attribute.</a> The attribute was updated from aria-multiselectable="true"` to data-allow-multiple.</li>
+</ul>
```

Comment:
This is a good case where we'd _want_ to consume a markdown file to display on site.

Might not be necessary to link to the CHANGELOG.md link, unless we plan to limit the number of items that we show. 

It'd be nice to see the changes in the sidenav too.

1. Do we plan to limit the number of items?
2. Do we want to include version numbers?
3. Can you show an example of a breaking change?